### PR TITLE
standardize unique_trade_id

### PR DESCRIPTION
**Pull request by soispoke**
Standardize descriptions and unique_trade_id columns + its associated tests across existing abstractions

I've checked that:

* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

**Review by soispoke — COMMENTED** [review]
dbt cloud is passing but was taking way too long + too much ressources

File: spellbook/models/magiceden/solana/magiceden_solana_schema.yml [hunk 1]
```diff
@@ -11,10 +11,8 @@ models:
     description: >
         MagicEden trades on Solana
     columns:
-      - name: unique_id
-        description:  "Unique id, combination of signatures[0] and id"
-        tests:
```
Comment (dot2dotseurat) [inline spellbook/models/magiceden/solana/magiceden_solana_schema.yml:16]: are we taking this out because it's taking too long to run this test?

**Review by soispoke — COMMENTED** [review]
Yes @dot2dotseurat I've added this in the previous comment but those tests have been running and passing dozens of time and take a loooot of compute/run time

**Comment by dot2dotseurat** [comment]
> Yes @dot2dotseurat I've added this in the previous comment but those tests have been running and passing dozens of time and take a loooot of compute/run time

alright, I think it's too complicated for right now but it would be cool if we could find a way to run these tests once a week. I think dbt core gives us enough tools to configure including and excluding tests but let's figure that out later. 

Like maybe we can use tags tests that run only for certain jobs.

**Review by dot2dotseurat — APPROVED** [review]

**Comment by soispoke** [comment]
Yeah I like the idea of running these tests once a week, I'll take a shot at it tomorrow :)

**Review by soispoke — COMMENTED** [review]
@dot2dotseurat Just so you know, I reverted the changes I made on tests here and will open a new PR to tackle this particular issue

**Pull request merged**